daemon/cmd/daemon_main.go

```diff
@@ -982,6 +982,10 @@ func init() {
 	flags.Bool(option.EnableCustomCallsName, false, "Enable tail call hooks for custom eBPF programs")
 	option.BindEnv(option.EnableCustomCallsName)
 
+	flags.Bool(option.NetfilterCompatibleMode, false, "In Netfilter Compatible mode, it guarantees that traffic will pass-through kernel netfilter and packets are routed based on IPtable rules. this mode may reduce network throughput. If not set, partially some traffic may pass-through kernel netfilter, but not guaranteed.")
```

Comment:
What about the following description?
```suggestion
	flags.Bool(option.NetfilterCompatibleMode, false, "If set to true, guarantees that traffic will pass through netfilter and that iptables rules will be enforced. This mode may reduce network throughput. If set to false (default), there are no guarantees: some traffic may skip netfilter.")
```

Reply:
Updated comments.